meetings/2023-09/september-26.md

```diff
@@ -39,908 +37,643 @@ You can find Abbreviations in delegates.txt
 
 SHN: Nothing more than what you’ve got right now. We’re going to have a press release to recognize the work that this community has been doing, and recognize the milestone. We are going to do a few special events, but maybe I can make a few more comments on the last day of the meeting because I don’t have them all ready now. I hope to see you all there. It will be excellent. Thank you.
 
-RPR: thanks especially to Roderigo for helping out to arrange that meeting. We can see you 
-in the room, Roderigo. Thank you. All right. So on to the usual items. So let’s start, 
-let’s see if we have approval for last -- actually, I don’t think we fully committed them yet, 
-so we should probably -- let’s return to that perhaps later in the meeting once we’ve got that. 
-All right, what about the current agenda?  Are there any objections to the current agenda?  And 
-I will say because I’m working on a laptop, I cannot see the element room right now. If any of 
-the other chairs do see anything, please just interrupt. I’m going to take silence as 
-approval. I will say with today’s current agenda, or this week’s current agenda, it is packed. 
-It is oversubscribed. You’ll see that we have many really good items in the overflow slot at 
-the moment. We’re going to work you know, we’re going to do our best to claw some back. 
-Maybe some people will consider deprioritizing items. I know I’ve already heard some some 
-candidates there that we may do. But the main thing is we need to really work hard to keep -- 
-to do food time keeping and stick to our time boxes. And with that, I think I can hand over to 
-Samina.
+RPR: thanks especially to Roderigo for helping out to arrange that meeting. We can see you in the room, Roderigo. Thank you. All right. So on to the usual items. So let’s start,
+let’s see if we have approval for last -- actually, I don’t think we fully committed them yet,
+so we should probably -- let’s return to that perhaps later in the meeting once we’ve got that.
+All right, what about the current agenda? Are there any objections to the current agenda? And I will say because I’m working on a laptop, I cannot see the element room right now. If any of the other chairs do see anything, please just interrupt. I’m going to take silence as approval. I will say with today’s current agenda, or this week’s current agenda, it is packed.
+It is oversubscribed. You’ll see that we have many really good items in the overflow slot at the moment. We’re going to work you know, we’re going to do our best to claw some back.
+Maybe some people will consider deprioritizing items. I know I’ve already heard some some candidates there that we may do. But the main thing is we need to really work hard to keep --
+to do food time keeping and stick to our time boxes. And with that, I think I can hand over to Samina.
```

Comment:
I don't understand what happened here -- how we have all these extra linebreaks after punctuation.  At first I thought maybe there was something wrong with the fix script, but I added this text, [as you can see in this PR](https://github.com/tc39/notes/pull/298), and the output is rendered as expected (one line, no extra linebreaks).  🤔  🤔  🤔 

This same problem occurs many times in this commit.

Reply:
update:  appears to be an interaction w/ the markdown fix followed by the linebreaks fix.  doing some additional testing...

Reply:
I think that test case works because it has extra trailing whitespace that I'm not seeing in this file.

Reply:
update: the md lint fixes removes the trailing whitespace from the lines.  which is great, but then the bad linebreaks fix does not do its job properly.  we perhaps could move the bad linebreaks fix to happen prior to the mdlint fix and maybe that resolves in the short term, but really the bad linebreaks script should handle the 2nd example below properly.

## original notes, which is currently fixed correctly by the linebreaks script

```
RPR: thanks especially to Roderigo for helping out to arrange that meeting. We can see you 
in the room, Roderigo. Thank you. All right. So on to the usual items. So let’s start, 
let’s see if we have approval for last -- actually, I don’t think we fully committed them yet, 
so we should probably -- let’s return to that perhaps later in the meeting once we’ve got that. 
All right, what about the current agenda?  Are there any objections to the current agenda?  And 
I will say because I’m working on a laptop, I cannot see the element room right now. If any of 
the other chairs do see anything, please just interrupt. I’m going to take silence as 
approval. I will say with today’s current agenda, or this week’s current agenda, it is packed. 
It is oversubscribed. You’ll see that we have many really good items in the overflow slot at 
the moment. We’re going to work you know, we’re going to do our best to claw some back. 
Maybe some people will consider deprioritizing items. I know I’ve already heard some some 
candidates there that we may do. But the main thing is we need to really work hard to keep -- 
to do food time keeping and stick to our time boxes. And with that, I think I can hand over to 
Samina.
```

## notes after markdownlint auto fix, which is NOT fixed correctly by the linebreaks script

```
RPR: thanks especially to Roderigo for helping out to arrange that meeting. We can see you
in the room, Roderigo. Thank you. All right. So on to the usual items. So let’s start,
let’s see if we have approval for last -- actually, I don’t think we fully committed them yet,
so we should probably -- let’s return to that perhaps later in the meeting once we’ve got that.
All right, what about the current agenda?  Are there any objections to the current agenda?  And
I will say because I’m working on a laptop, I cannot see the element room right now. If any of
the other chairs do see anything, please just interrupt. I’m going to take silence as
approval. I will say with today’s current agenda, or this week’s current agenda, it is packed.
It is oversubscribed. You’ll see that we have many really good items in the overflow slot at
the moment. We’re going to work you know, we’re going to do our best to claw some back.
Maybe some people will consider deprioritizing items. I know I’ve already heard some some
candidates there that we may do. But the main thing is we need to really work hard to keep --
to do food time keeping and stick to our time boxes. And with that, I think I can hand over to
Samina.
```

Reply:
alright, so this is fixed in the aforementioned PR and should both detect and resolve these issues

Reply:
fixes are here:  https://github.com/SaminaHusain/notes/pull/1

Reply:
PR was merged, this PR rebased, and fix commit added